data/card_catalogue/social.json

```diff
@@ -125,7 +125,7 @@
         "active": "Change your appearance to match that of another exactly for as long as you'd like.",
         "passive": null
       },
-      "flavour": "\"I'm Revins, Sniver went into the other room to get his homework.\" - Little Sniver",
```

Comment:
This one isn't that good on it's own, so I've replaced it with something more basic.